libs/sdk-py/langgraph_sdk/routing/__init__.py

```diff
@@ -0,0 +1,111 @@
+import typing
+
+from langgraph_sdk.routing import types
+from langgraph_sdk.routing.types import Middleware
+
+
+@typing.final
+class Router:
```

Comment:
This looks like a container for 3 independent things b/c there's no api router concept in starlette. 

It may be helpful to restructure the first documentation paragraph visually, so it's easier to see that the Middleware has global scope. Folks may have the similar confusion about the lifespan (although less likely to be an issue). -- content is good, i think only visual layout (e.g., markdown bullet point)

Reply:
(had somem updates so we're goingt o directly accept a starlette app)

Reply:
There is a Starlette router but it constructs the middleware at init time so we wouldn't be able to extract the relevant bits and put it in our router or apply the middleware to the other routes (which imo is expected by the user) without it gettting a tad fragile